docs/en/enterprise-edition/content-collections/administration/action-plans.adoc

```diff
@@ -0,0 +1,63 @@
+Action plans are generated based on the severity of your alerts, and are only available if high/critical alerts are available.
+
+.. *Assigned to Me*— Lists all the *Action Plans* assigned to you for resolution. Click on the *Assignee* filter to reassign any selected Action Plan.
+
+.. *Unassigned Action Plans*— Lists *Action Plans* that do not currently have an owner. Click on the *Assignee* filter to select an owner.
+
+
+=== Action Plan Workflows 
+
+From the Action Plan dashboard select any *Action Plan* to get a closer look at the alerts addressed by the plan, as well as the number of impacted assets.
+Each action plan helps group alerts to help burn them down more efficiently. Alerts are logically grouped together so that a single action on one asset can remove the corresponding alerts. 
+
+image::administration/action-plans-2.gif[]
+
+The tags, as indicated in the image above, consolidate the different finding types grouped by the alerts. You can also select any Alert to further investigate the root cause. Learn more about remediating xref:../alerts/view-respond-to-prisma-cloud-alerts.adoc[Alerts].  
+
+Action Plans comprise of asset specific details to help security teams rapidly remediate the risk. Sections include:
+
+* *Overview*— Provides a summary of the action plan finding, the primary asset affected, as well as the ability to trace through the impact of the grouped issues by viewing attack paths, alerts, vulnerabilities, and findings. 
+ 
+* *How to Fix*— Leverages machine learning and large language models to combine issues across several alerts that affect an asset to provides step by step recommendations to remediate the risk.
+For example, an Action Plan can include specific steps to address an issue with an identity role that needs to be updated to resolve several alerts. You can also forward the issue to internal security teams for further resolution using one of the options below:
+
+** Select the *Create a JIRA* ticket button to automatically create a Jira ticket using a standard template. If the correct template is not visible, you can create one by clicking on the highlighted link.
+This template can then be accessed internally in Prisma Cloud. 
+
+** Select *Send a Slack message* to reach out to issue owners via Slack to ensure proper visibility to the relevant group or individuals as needed. 
+You can select a Slack group as well as enter an optional message to be sent alongside the action plan.
+
+* *Related Alerts*— List all the alerts associated with the action plan. Click on any alert to view Attack Paths, Vulnerabilities and other related details pertaining to the alert. This view is highly customizable and alerts can be individually addressed or snoozed from the action plan.
```

Comment:
Suggest re-wording the sentence: 
Related Alerts— List all the alerts associated with the action plan. Click on any alert to view Attack Paths, Vulnerabilities, and other details related to the alert.